spec/rails_helper.rb

```diff
@@ -52,3 +52,10 @@
 
   config.global_fixtures = :all
 end
+
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails
+  end
+end
```

Comment:
@NikitaAvvakumov what do you think about adding the config to **spec/support/shoulda_matchers.rb**  ? I saw in many Rails projects that developers tend to put RSpec additional config in **spec/support** folder